content/docs/tasks/telemetry/using-istio-dashboard/index.md

```diff
@@ -70,43 +73,73 @@ the example application throughout this task.
+
+1.  Visualize Service Dashboards.
+
+    From the Grafana dashboard's left hand corner navigation menu, you can navigate to Istio Service Dashboard or visit
+    [http://localhost:3000/dashboard/db/istio-service-dashboard](http://localhost:3000/dashboard/db/istio-service-dashboard) in your web browser.
+
+    The Istio Service Dashboard will look similar to:
+
+    {{< image width="100%" ratio="51.65%"
+    link="./istio-service-dashboard.png"
+    caption="Istio Service Dashboard"
+    >}}
+
+    This gives details about metrics for the service and then client workloads(workloads that are calling this service)
+    and service workloads(workloads that are providing this service) for that service.
+
+1.  Visualize Workload Dashboards.
+
+    From the Grafana dashboard's left hand corner navigation menu, you can navigate to Istio Workload Dashboard or visit
+    [http://localhost:3000/dashboard/db/istio-workload-dashboard](http://localhost:3000/dashboard/db/istio-workload-dashboard) in your web browser.
+
+    The Istio Workload Dashboard will look similar to:
+
+    {{< image width="100%" ratio="52.68%"
+    link="./istio-workload-dashboard.png"
+    caption="Istio Workload Dashboard"
+    >}}
+
+    This gives details about metrics for each workload and then inbound workloads and outbound services for that workload.
 
 ### About the Grafana add-on
 
 The Grafana add-on is a preconfigured instance of Grafana. The base image
-([`grafana/grafana:4.1.2`](https://hub.docker.com/r/grafana/grafana/)) has been
+([`grafana/grafana:5.0.4`](https://hub.docker.com/r/grafana/grafana/)) has been
 modified to start with both a Prometheus data source and the Istio Dashboard
 installed. The base install files for Istio, and Mixer in particular, ship with
 a default configuration of global (used for every service) metrics. The Istio
 Dashboard is built to be used in conjunction with the default Istio metrics
 configuration and a Prometheus backend.
 
 The Istio Dashboard consists of three main sections:
-1. A Global Summary View. This section provides high-level summary of HTTP
-   requests flowing through the service mesh.
-1. A Mesh Summary View. This section provides slightly more detail than the
-   Global Summary View, allowing per-service filtering and selection.
+
+1. A Mesh Summary View. This section provides Global Summary view of the Istio Mesh and shows HTTP/gRPC and TCP
+   workloads in Istio Mesh.
+
 1. Individual Services View. This section provides metrics about requests and
-   responses for each individual service within the mesh (HTTP and TCP).
+   responses for each individual service within the mesh (HTTP/gRPC and TCP).
+   Also, give metrics about client and service workloads for this service.
+
+1. Individual Workloads View: This section provides metrics about requests and
+   responses for each individual workloads within the mesh (HTTP/gRPC and TCP).
+   Also, give metrics about inbound workloads and outbound services for this workload.
```

Comment:
"inbound workloads" and "outbound services" doesn't quite work. Workloads and services aren't directional. Do you mean inbound and outbound traffic?

Reply:
Added above explanation in Visualize Workload Dashboards section